glean_parser/schemas/metrics.1-0-0.schema.yaml

```diff
@@ -239,16 +239,6 @@ definitions:
         description: |
           **Required.**
 
-          May be one of the following values:
```

Comment:
I still think we should document the default behavior here.  Maybe retain this, but add a note that the default behavior might be overridden in certain applications ... see the `custom_validate_expires` config.

Reply:
Good idea.